CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+# 76.1.0
+
+Add additional snapshot tests and mark outdated theming API as deprecated.
+
+## New deprecations
+
+The previous theming API is being deprecated in favor of MDCContainerScheme.
```

Comment:
In favor of Theming Extensions? Maybe link to the docs on GitHub?

Reply:
Docs here: 
https://github.com/material-components/material-components-ios/blob/develop/components/schemes/README.md

Reply:
Why wasn't this feedback addressed before merging?

Reply:
Follow-up PR incoming.